exercises/acronym/acronym_test.go

```diff
@@ -0,0 +1,47 @@
+package acronym
+
+import (
+	"testing"
+)
+
+const targetTestVersion = 1
+
+type testCase struct {
+	input     string
+	expected  string
+	expectErr bool
+}
+
+var stringTestCases = []testCase{
+	{"Portable Network Graphics", "PNG", false},
+	{"HyperText Markup Language", "HTML", false},
+	{"Ruby on Rails", "ROR", false},
+	{"PHP: Hypertext Preprocessor", "PHP", false},
+	{"First In, First Out", "FIFO", false},
+	{"Complementary metal-oxide semiconductor", "CMOS", false},
```

Comment:
I could not think any negative testcase at time of writing so kept it for future development.